tsl/test/expected/remote_txn.out

```diff
@@ -553,14 +735,40 @@ NOTICE:  [loopback]:  PREPARE prep_1 AS SELECT 1
  
 (1 row)
 
+        --connection in transaction state
```

Comment:
In this test case I would add some data modification commands before `SAVEPOINT save_1;` and make sure those are visible after the commit. That's a much more surefire correctness check.